lib/DefaultClient.php

```diff
@@ -491,6 +492,8 @@ private function doWrite(RequestCycle $requestCycle) {
             }
 
             yield from $this->doRead($requestCycle, $socket, $connectionInfo);
+        } catch (StreamException $e) {
+            throw new SocketException("Failed to write to byte-stream", 0, $e);
```

Comment:
The catch is around the `write` and `doRead`, should the message really say `write`?

Reply:
Duh, probably not. I'll fix this later, I'm currently on mobile only.